src/clients/projectStorage.js

```diff
@@ -0,0 +1,20 @@
+const PROJECT_STORAGE_KEY = 'last-closed-session-project-state';
```

Comment:
This looks good structurally, but I’d prefer to follow the pattern of naming the client module after the service it wraps rather than the purpose it’s currently serving (e.g. other clients are `firebase`, `gitHub`, etc.). So probably just `src/clients/localStorage.js`